internal/config/default.yaml

```diff
@@ -2,4 +2,7 @@ server:
   port: 80
 
 logger:
-  level: info
\ No newline at end of file
+  level: info
+
+rabbitmq:
+  url: amqp://localhost/
```

Comment:
I also think about it. And I think that is the best approach because the method that connects to RabbitMQ requires a URL and IMO there is no reason to require the construct have many arguments (hostname, port, user, password, vhost) to format a URL. 
Also, the URL `amqp://localhost` has the default settings, it is the same as `amqp://guest:guest@localhost:5672/`